### DISCUSSION TITLE
Is there a document that lists best practices to follow while using CRI-O

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#7848] [Q&A]
As there is hardening guideline from CIS for Docker, is there some hardening guideline for CRI-O or may be some document that lists best practices to be followed while using CRI-O?
It can have list like:
- permission of conf files?
- any setting that should not be assigned particular value

or any other guideline

**Comment by alpana17** [comment]
@kwilczynski Here, I have started discussion

**Comment by haircommander** [comment]
We generally like to think our OOTB configuration is a good baseline configuration for security. I'd also recommend making sure your /etc/containers/registries.conf doesn't enable any insecure registries (force the use of HTTPS). Lots of our options have equivalents in docker, so if there are specific recommendations CIS makes for docker it's likely there are equivalents in cri-o

**Comment by alpana17** [comment]
Also, how does cri-o impacted by selinux setting?
